src/toil/job.py

```diff
@@ -779,14 +778,19 @@ def makeString(x: Union[str, bytes, None]) -> str:
         # ID of this job description in the JobStore.
         self.jobStoreID: Union[str, TemporaryID] = TemporaryID()
 
-        # Mostly fake, not-really-executable command string that encodes how to
-        # find the Job body data that this JobDescription describes, and the
-        # module(s) needed to unpickle it.
+        # Actual command string we intend for the batch system to run somewhere
+        # Used internally by the leader.
+        # TODO: Really internal leader state!
+        self._worker_command: Optional[str] = None
+
+        # String that encodes how to find the Job body data that this
+        # JobDescription describes, and the module(s) needed to unpickle it.
+        # None if no body needs to run.
         #
-        # Gets replaced with/rewritten into the real, executable command when
-        # the leader passes the description off to the batch system to be
-        # executed.
-        self.command: Optional[str] = command
+        # Used to be the "command" until it was extended beyond making sense as
+        # an actual command, so now it's just a string with several
+        # space-separated fields. See attach_body() and get_body().
```

Comment:
I'm not sure this comment is as useful as just describing the contents of `self._body_spec` as a space-separated string of filestore_id and command.  Would suggest changing these three lines to ` # a space-separated string of filestore_id and command` .